setup.sh

```diff
@@ -14,4 +14,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Can we `set -e` this script? So if there is an error in the code, it propagates to the top level error status?

Reply:
Done